.docker/odoo/Dockerfile

```diff
@@ -12,13 +12,10 @@ RUN apt update; \
       tee /etc/apt/sources.list.d/docker.list > /dev/null; \
     apt update; \
     apt -y --no-install-recommends install docker-ce; \
-    useradd -m runbot; \
-    chown -R runbot:runbot /var/lib/nginx; \
-    chown -R runbot:runbot /var/log/nginx;
+    useradd -m runbot;
 
 USER runbot
 RUN mkdir -p ~/instance; \
-    git clone -b 11.0 --depth=1  https://github.com/Vauxoo/runbot-addons.git ~/instance/dependencies/runbot-addons; \
```

Comment:
I prefer the old option

Since that it could have the whole packages installed and cloned without run extra steps locally

(for devops it is easy)

Even you can develop using vscode + remote container in order to use the source from the container

Reply:
...even if the repository cloned is the same than this one

We could use directly a `docker push` to quay or docker hub

Reply:
I think a build arg could be a nice idea to either use a local repo or setup odoo for development, however this is an old project and hopefully one day we won't even develop on it haha so I will revert it to focus on the bot itself